packages/netlify-cms-core/src/components/Collection/Sidebar.js

```diff
@@ -35,7 +35,7 @@ const SidebarHeading = styled.h2`
 const SearchContainer = styled.div`
   display: flex;
   align-items: center;
-  margin: 0 8px;
+  margin: 0 12px;
```

Comment:
Added this in because the search field didn't align entirely with the title and items